app/editor/src/features/admin/filters/FilterFormImportExport.tsx

```diff
@@ -0,0 +1,114 @@
+import 'prismjs/themes/prism.css';
+import 'prismjs/components/prism-csharp';
+import 'prismjs/components/prism-cshtml';
+import 'prismjs/components/prism-json';
+
+import { useFormikContext } from 'formik';
+import { highlight, languages } from 'prismjs';
+import React from 'react';
+import Editor from 'react-simple-code-editor';
+import { useLookupOptions } from 'store/hooks';
+import {
+  Button,
+  ButtonVariant,
+  Col,
+  generateQuery,
+  IFilterModel,
+  IFilterSettingsModel,
+  Row,
+} from 'tno-core';
+
+import { exportFilter, parseExportedFilter } from './utils';
+
+/**
+ * The page used to view and edit report filter.
+ * @returns Component.
+ */
+export const FilterFormImportExport: React.FC = () => {
+  const { values, setFieldValue } = useFormikContext<IFilterModel>();
+  const [{ series, products, sources, contributors, actions }] = useLookupOptions();
+
+  const [rawFilter, setRawFilter] = React.useState('{}');
+
+  const parseImport = React.useCallback(
+    (rawExportedFilter: any) => {
+      var importedFilter = parseExportedFilter(
+        rawExportedFilter,
+        actions,
+        contributors,
+        series,
+        sources,
+        products,
+      );
+
+      setFieldValue('name', importedFilter.name ?? '');
```

Comment:
Instead of three calls just do one.

`setValues({...importedFilter})`

Reply:
cant find that method...?